content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -9,6 +9,7 @@
     font-weight: bold;
     margin: 0;
 
+    &.grey-title,
     span {
```

Comment:
Is the span selector used for anything now?

Reply:
This is used once more in the Search Dialog component. Deleting it in the next PR

Reply:
Ah right, this is collapsible section styles not just top sites.